### ISSUE TITLE
[Feature Request] Quicker way to remove paper from folder without deleting.

### ISSUE THREAD

**Issue by pratch** [Future-Scholars/paperlib#597]
Currently, if I want to remove a paper from a folder, I need to do these cumbersome actions: 

right click > remove from > folders > folder_name

If I am already in a folder and click "remove" on a paper, there shouldn't need to be the option to select other folders. 

Even better, maybe change pressing DEL key on a paper while in a folder to removing and not deleting by default? There's already an easy way to delete that is the trash icon above.

**Comment by GeoffreyChen777** [comment]
You misunderstand, what is `remove from`, and what is `delete`.

- `remove from`: untag a paper, or remove a paper from a folder.
- `delete` and the menu button: delete the paper from the paperlib library and delete its files on the disk.

They are totally different.

In paperlib, all papers exist in the library, users can assign different tags for them, and also can assign different folders for them. Yes, a folder is just a special tag.

The main UI is showing the data in the library, when you click different tags, folders, it's doing a filtering. The papers satisfying the filtering query will show up.

**Comment by GeoffreyChen777** [comment]
To remove a paper from a folder/tag, you can click this:
![image](https://github.com/user-attachments/assets/f9b628d0-7db7-42d9-a3c3-9755f4c3fab0)

**Comment by pratch** [comment]
Thanks, clicking on the x does make it faster. I still think DEL key on paper should default to untag the folder, essentially removing the paper from that folder, instead of hard deleting though. That is the default behavior in zotero (DEL to remove from collection, not deleting the PDF and everything), so people are probably used to this behavior.

**Comment by GeoffreyChen777** [comment]
Yes, that is the software habit you develop in Zotero, but Paperlib is not Zotero. Paperlib and Zotero have many different design concepts. Zotero uses folders to organise papers. However, we use views of query sentences. You can look at the smart filter feature of Paperlib. It will let you understand Paperlib deeper.

https://paperlib.app/en/doc/smart-filter/

If we copy Zotero, why not use Zotero directly?  So try to get used to Paperlib. I believe you will make it really quick.

**Comment by pratch** [comment]
I did not intend to say just copy zotero, but in this specific case, the way DEL works in zotero is more intuitive. If you are in a folder (I emphasize again that this is in a specific folder view, not "All Papers" view) and you press DEL, your intention is most likely to untag that folder, not deleting the ref which will remove it from other folders as well.

**Comment by GeoffreyChen777** [comment]
Then, one single button has two different meanings, depending on the context. It's confusing.

With your design, in a folder view, if I want to delete the paper from my library, there will be no way to achieve that.

**Comment by GeoffreyChen777** [comment]
Paperlib's such a design has been used for more than 4 years, we cannot change it, as many user have been used to it.

**Comment by pratch** [comment]
You can achieve that by clicking on the trash icon above, or right click > delete. My argument is that removing from folder is more common than deleting, so having simple shortcut for it makes sense. Maybe if you don't want DEL key to mean 2 different things, how about BACKSPACE to untag folder then?

**Comment by GeoffreyChen777** [comment]
However, one paper can be in several folders. How to solve this issue? untag all folders when pressing backspace?

**Comment by pratch** [comment]
When you press backspace on a paper while you are in folder X, untag X.

backspace while in "All papers" does nothing.

**Comment by GeoffreyChen777** [comment]
So, your request is just a hotkey, to move the selected paper out of the current selected folder/tag.

Currently please click the `x` in the detail panel to achieve this.

It makes sense but is not easy to implement.

**Comment by pratch** [comment]
No problem. I understand the need to keep things simple. The x method will do for me. Thanks for the quick replies I really appreciate it. I love paperlib very much.
